CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 All notable changes to this project will be documented in this file. This change log follows the conventions of [keepachangelog.com](http://keepachangelog.com/).
 
 ## [Unreleased]
+
 ### Changed
+
 - Add a new arity to `make-widget-async` to provide a different widget shape.
 
+### Added
+
+- A new `CredentialsProvider` that supports `credential_process`
+
+
 ## [0.1.1] - 2021-10-11
 ### Changed
 - Documentation on how to make the widgets.
```

Comment:
This previous changelog was a nonsense placeholder, so we should probably delete it here since you're the first entry.

Reply:
Will do, thanks.

Reply:
Addressed via b0a6175957fa33bc6b830a7169a5661315077a37 — thanks!